mlflow/genai/judges/judge_trace_utils.py

```diff
@@ -0,0 +1,52 @@
+"""Utilities for extracting data from MLflow traces."""
+
+from mlflow.entities.trace import Trace
+from mlflow.genai.utils.trace_utils import parse_inputs_to_str, parse_outputs_to_str
+
+
+def extract_request_from_trace(trace: Trace) -> str:
```

Comment:
```suggestion
def extract_request_from_trace(trace: Trace) -> Optional[str]:
```

Instead of returning an empty string from this utility function to indicate an absent request, can we return `None`?

Callers can then impute missing values as necessary